drivers/i2c/i2c_mcux.c

```diff
@@ -186,15 +186,15 @@ static const struct i2c_driver_api i2c_mcux_driver_api = {
 static void i2c_mcux_config_func_0(struct device *dev);
 
 static const struct i2c_mcux_config i2c_mcux_config_0 = {
-	.base = I2C0,
+	.base = (void *)CONFIG_I2C_MCUX_0_BASE_ADDRESS,
```

Comment:
Thanks, missed the I2C1 case.  Will fix this up and split it out from the I2C DTS cfg patch for now.